client/ayon_houdini/plugins/create/create_FBX.py

```diff
@@ -175,3 +190,15 @@ def get_selection(self):
             )
 
         return selection
+    
```

Comment:
Cosmetics (code style says we should have two line breaks between top level classes)
```suggestion


```